fuzzers/fuzz_picosha2_hex_string.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
```

Comment:
The std::cout stuff is very helpful for debugging, but it will tend to hurt performance during fuzzing (any disk IO will). Could you please update this harness to omit disk IO like `std::cout`. You could probably just omit the `CalcAndOutput()` function entirely and call `picosha2::hash256_hex_string()` directly.